src/main/java/com/org/jenkins/custom/jenkins/distribution/service/GithubJwtHelper.java

```diff
@@ -0,0 +1,50 @@
+package com.org.jenkins.custom.jenkins.distribution.service;
+
+import io.jsonwebtoken.JwtBuilder;
+import io.jsonwebtoken.SignatureAlgorithm;
+import io.jsonwebtoken.Jwts;
+import java.io.File;
+import java.security.Key;
+import java.security.KeyFactory;
+import java.security.PrivateKey;
+import java.security.spec.PKCS8EncodedKeySpec;
+import java.util.Date;
+import com.google.common.io.Files;
+
+
+public class GithubJwtHelper {
+
+    static PrivateKey get(String filename) throws Exception {
```

Comment:
Correct me if I am wrong. I think this method 1) encodes the key, then 2) it encrypts it. Is that right?

Reply:
Absolutely :)

Reply:
Ok, then this explanation can be put in the javadoc.